packages/orchestration/test/examples/stake-atom.contract.test.ts

```diff
@@ -18,51 +19,107 @@ const startContract = async ({
   timer,
   marshaller,
   storage,
-  bld,
+  issuerKeywordRecord,
+  terms = {
+    chainId: 'cosmoshub-4',
+    hostConnectionId: 'connection-1',
+    controllerConnectionId: 'connection-2',
+    bondDenom: 'uatom',
+    icqEnabled: false,
+  } as StakeIcaTerms,
 }) => {
   const { zoe, bundleAndInstall } = await setUpZoeForTest();
   const installation: Installation<StartFn> =
     await bundleAndInstall(contractFile);
 
   const { publicFacet } = await E(zoe).startInstance(
     installation,
-    { In: bld.issuer },
-    {
-      chainId: 'cosmoshub-4',
-      hostConnectionId: 'connection-1',
-      controllerConnectionId: 'connection-2',
-      bondDenom: 'uatom',
-    },
+    issuerKeywordRecord,
+    terms,
     {
       marshaller,
       orchestration,
-      storageNode: storage.rootNode,
+      storageNode: storage.rootNode.makeChildNode('stakeAtom'),
       timer,
     },
   );
   return { publicFacet, zoe };
 };
 
-test('makeAccount, deposit, withdraw', async t => {
+test('makeAccount, getAddress, getBalances, getBalance', async t => {
   const {
     bootstrap,
     brands: { ist },
     utils,
   } = await commonSetup(t);
-  const { publicFacet } = await startContract({ ...bootstrap, bld: ist });
+  const { publicFacet } = await startContract({
+    ...bootstrap,
+    issuerKeywordRecord: { In: ist.issuer },
+  });
 
   t.log('make an ICA account');
   const account = await E(publicFacet).makeAccount();
   t.truthy(account, 'account is returned');
-  const address = await E(account).getAddress();
-  // XXX address.address is weird
-  //   t.regex(address.address, /agoric1/);
-  t.like(address, { chainId: 'cosmoshub-4', addressEncoding: 'bech32' });
+  const chainAddress = await E(account).getAddress();
+  // t.regex(address.address, /cosmos1/);
+  t.like(chainAddress, { chainId: 'cosmoshub-4', addressEncoding: 'bech32' });
 
   t.log('deposit 100 bld to account');
   await E(account).deposit(await utils.pourPayment(ist.units(100)));
 
   await t.throwsAsync(E(account).getBalances(), {
     message: 'not yet implemented',
   });
+
+  await t.throwsAsync(E(account).getBalance('uatom'), {
+    message: 'Queries not enabled.',
+  });
+});
+
+test('makeAccountInvitationMaker', async t => {
+  const {
+    bootstrap,
+    brands: { ist },
+  } = await commonSetup(t);
+  const { publicFacet, zoe } = await startContract({
+    ...bootstrap,
+    issuerKeywordRecord: { In: ist.issuer },
+  });
+  const inv = await E(publicFacet).makeAccountInvitationMaker();
+  t.log('make an offer for ICA account');
+  t.log('inv', inv);
+
+  const seat = await E(zoe).offer(inv);
+  t.log('seat', seat);
```

Comment:
logging values is useful when diagnosing a failure but noise when the tests are passing.